examples/md-flexible/scripts/plotDiffPredictionTest.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python3
+
+import sys
+import plotly.graph_objects as go
+import re
+from numpy import double
+
+# THIS SCRIPT NEEDS AT LEAST PYTHON 3.8.
+# However, lesser version will probably fail due to invalid syntax instead of this assertion
+
+# ---------------------------------------------- Input ----------------------------------------------
+
+# help message
+for arg in sys.argv[1:]:
+    if "--help" in arg:
+        print("Usage: ./plotDiffPredictionTest.py [output option] [path/To/mdFlex/std.out ...]")
```

Comment:
```suggestion
        print("Usage: ./plotDiffPredictionTest.py OPTION path/To/mdFlex/std.out ...")
```
Non-optional arguments don't get brackets ;)